src/WpfMath/TexFormulaParser.cs

```diff
@@ -383,7 +383,7 @@ internal static SourceSpan ReadElement(SourceSpan value, ref int position)
             Parse(ReadElement(value, ref position), formula.TextStyle, environment.CreateChildEnvironment());
 
         /// <remarks>May return <c>null</c> for commands that produce no atoms.</remarks>
-        private Atom ProcessCommand(
+        private (AtomAppendMode, Atom) ProcessCommand(
```

Comment:
Due to obscure restrictions, we no more able to use value tuples in our code :(

This has to be rewritten to `Tuple<AtomAppendMode, Atom>`.